.github/workflows/build.yml

```diff
@@ -86,3 +88,22 @@ jobs:
           if-no-files-found: ignore
           path: |
             /tmp/sphinx-*.log
+
+  release:
+    needs: [build]
+    if: startsWith(github.ref, 'refs/tags/v')
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/download-artifact@v2
+        with:
+          name: jupyterlite dist ${{ github.run_number }}
+          path: ./dist
+      - name: Release
+        uses: softprops/action-gh-release@v1
+        with:
+          draft: true
+          prerelease: true
+          files: |
+            dist/jupyterlite-*.tgz
```

Comment:
any particular reason to not ship _evrything_, including the SHA256?

Reply:
No real reason . Let's change it to `dist/*` :+1:

Reply:
```suggestion
            dist/*
```